docs/intro-react.md

```diff
@@ -0,0 +1,613 @@
+
+You can add custom pet pictures to your component by:
+
+- adding an `Image` component to your `MyPet` component
+- setting `Image`‘s `source` via `MyPet`‘s `props` with `{props.picURI}`
+- then passing `picURI` values in each `MyPet`
+
+<div class="toggler">
+  <ul role="tablist" id="toggle-syntax">
+    <li id="functional" class="button-functional" aria-selected="false" role="tab" tabindex="0" aria-controls="functionaltab" onclick="displayTab('syntax', 'functional')">
+      Functional Syntax
+    </li>
+    <li id="classical" class="button-classical" aria-selected="false" role="tab" tabindex="0" aria-controls="classicaltab" onclick="displayTab('syntax', 'classical')">
+      Class Syntax
+    </li>
+  </ul>
+</div>
+
+<block class="functional webNote androidNote" />
+
+```SnackPlayer name=Custom%20Pictures
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+
+function MyPet(props) {
+  return (
+    <View>
+      <Image source={props.picURI} style={{width: 200, height: 200}}/>
+      <Text>Hello, I am {props.name}!</Text>
+    </View>
+  );
+}
+
+export default function MyPets() {
+    return (
+      <View>
+        <MyPet name="Peebo" picURI="https://placekitten.com/g/200/200" />
+        <MyPet name="Henrietta" picURI="https://placekitten.com/g/200/200" />
+        <MyPet name="Tenno" picURI="https://placekitten.com/g/200/200" />
+      </View>
+    );
+}
+```
+
+<block class="classical webNote androidNote" />
+
+```SnackPlayer name=Custom%20Pictures
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+
+export class MyPet extends Component {
+    render(){
+        return (
+            <View>
+                <Image source={this.props.picURI} style={{width: 200, height: 200}}/>
+                <Text>Hello, I am {this.props.name}!</Text>
+            </View>
+        );
+    }
+}
+
+export default class MyPets extends Component {
+    render(){
+        return (
+            <View>
+                <MyPet name="Peebo" picURI="https://placekitten.com/g/200/200" />
+                <MyPet name="Henrietta" picURI="https://placekitten.com/g/200/200" />
+                <MyPet name="Tenno" picURI="https://placekitten.com/g/200/200" />
+            </View>
+        );
+    }
+}
+```
+
+<block class="functional classical webNote androidNote" />
+
+You can build many things with `props` and the Core Components [`Text`](text), [`Image`](image), and [`View`](view)! But to build something interactive, you’ll need `state`.
+
+## State
```

Comment:
do you think it's worth adding a class-based state explanation as well? might trip folks up if they follow that through the doc and get to the end to find only the hooks approach to state

Reply:
After some user testing and discussion with Dan and a few others, it seems the Best Thing might be removing the class examples and sharing a legacy example and explanation at the end of the intro.

Reply:
that seems reasonable!

Reply:
In the end, we did offer a class-based state example!